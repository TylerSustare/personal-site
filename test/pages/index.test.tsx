import React from 'react';
import {
  render,
  // , fireEvent
} from '../testUtils';
import HomeContent from '../../src/home/index';

describe('Home page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<HomeContent />, {});
    expect(asFragment()).toMatchSnapshot();
  });

  // it('clicking button triggers alert', () => {
  //   const { getByText } = render(<Home />, {});
  //   window.alert = jest.fn();
  //   fireEvent.click(getByText('Test Button'));
  //   expect(window.alert).toHaveBeenCalledWith('With typescript and Jest');
  // });
});
