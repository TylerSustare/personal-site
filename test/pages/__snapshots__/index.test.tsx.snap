// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-root-1"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <div>
          <button
            aria-label="menu"
            class="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-4 MuiIconButton-colorInherit MuiIconButton-edgeStart"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiIconButton-label"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          class="MuiContainer-root MuiContainer-maxWidthMd"
        >
          <div
            class="MuiBox-root MuiBox-root-5"
          >
            <h5
              class="MuiTypography-root MuiTypography-h5"
            >
              Welcome
            </h5>
          </div>
        </div>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Login
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </header>
  </div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthMd"
  >
    <div
      class="MuiBox-root MuiBox-root-6"
    >
      <div
        class="MuiBox-root MuiBox-root-7"
      >
        <h1
          class="MuiTypography-root MuiTypography-h2 MuiTypography-gutterBottom"
        >
          👋 Hi, I'm Tyler
        </h1>
        <div
          class="MuiBox-root MuiBox-root-8"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
          >
            I like to do coding stuff and spend time with my family. 👨‍👩‍👧‍👦
            <br />
            Here is the important stuff about the people I love, my family. If you would like to see more about me professionally go here
            <a
              aria-disabled="false"
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary"
              href="/about"
              role="button"
              tabindex="0"
            >
              <span
                class="MuiButton-label"
              >
                Professional About Me
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </p>
        </div>
        <div
          class="MuiBox-root MuiBox-root-9"
        >
          <h2
            class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom"
          >
            👩‍🏫/🐰Bunny
          </h2>
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
          >
            My wife of 7 years, and middle school sweetheart is the backbone of our home, and the reason for any and all of the successes we've had as a team. She's the greatest wife and mommy on the planet and I will absolutely fight you over that.
          </p>
        </div>
        <div
          class="MuiBox-root MuiBox-root-10"
        >
          <h2
            class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom"
          >
            👧 Smidge
          </h2>
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
          >
            Sweetest and most creative little girl in the whole world.
          </p>
        </div>
        <div
          class="MuiBox-root MuiBox-root-11"
        >
          <h2
            class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom"
          >
            👶 Denny
          </h2>
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
          >
            Such a kind and thoughtful little guy.
          </p>
        </div>
        <p
          class="MuiTypography-root makeStyles-root-12 MuiTypography-body1 MuiTypography-colorTextSecondary"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root makeStyles-lightBulb-13"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M9 21c0 .55.45 1 1 1h4c.55 0 1-.45 1-1v-1H9v1zm3-19C8.14 2 5 5.14 5 9c0 2.38 1.19 4.47 3 5.74V17c0 .55.45 1 1 1h6c.55 0 1-.45 1-1v-2.26c1.81-1.27 3-3.36 3-5.74 0-3.86-3.14-7-7-7zm2.85 11.1l-.85.6V16h-4v-2.3l-.85-.6C7.8 12.16 7 10.63 7 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 1.63-.8 3.16-2.15 4.1z"
            />
          </svg>
          Pro tip: See more 
          <a
            class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
            href="https://material-ui.com/getting-started/templates/"
          >
            templates
          </a>
           on the Material-UI documentation.
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
        >
          Copyright © 
          <a
            class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
            href="https://material-ui.com/"
          >
            Tyler Sustare
          </a>
           2020.
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
